charts/fission-all/values.yaml

```diff
@@ -99,3 +99,8 @@ pruneInterval: 60
 
 ## Fission pre-install/pre-upgrade checks live in this image
 preUpgradeChecksImage: fission/pre-upgrade-checks
+
+## the environment where fission is installed.
+## If its set to any other value other than production, then debug information is returned in http response for
+## fn test cli in case of fission internal errors.
+envType: production
```

Comment:
I think the production/debug flag should be a per-function. Ok to have a global default, though.

Reply:
^ agreed - since " any other value other than production, then debug information is returned", maybe it is clearer to just make it a boolean debug flag `debug: true` or `env.debug: true`. Or do we plan on having more env types?

Reply:
I was thinking that there's one installation of fission per each env - dev, qa, production etc.
However, if there is a possibility that one fission instance can handle a few functions that are dev grade and few that are production grade, then it makes sense to have a per function debug flag.